src/Http/Controllers/U2fController.php

```diff
@@ -35,90 +29,83 @@ public function __construct(LaravelU2f $u2f, Config $config)
     }
 
     /**
-     * @author LAHAXE Arnaud
```

Comment:
😞

Reply:
I thought you guys would just make the commit for the author....
Just added after 7 months seeing no extra commit figuring I would have to do it myself in this case.